### PR TITLE
(GH-1401) Added initial call to Chocolatey after installation

### LINKED ISSUE #1401: installation/setup: run choco once to initialize the config file
### What You Are Seeing?

First time after installing Chocolatey as admin, I start a non-admin command prompt, run `choco pack` and see this error:
image

### What is Expected?
It should work fine (whatever Chocolatey wants to write to ProgramData needs to be written at install time). Alternatively, consider saving the config file to LocalAppData (this doesn't need admin permissions to write, and is generally a better thing to do. Program Data goes to one place, configs, logs etc. go to another).

### Output Log
Unfortunately I didn't grab the output log other than the screenshot above. I tried a workaround (which is to run pack under admin, and that created the config file and I got unblocked).

### PR DESCRIPTION
**Pull request by gep13**
Fixes #1401

### CODE REVIEW

File: nuget/chocolatey/tools/chocolateysetup.psm1 [hunk 1]
```diff
@@ -700,4 +701,17 @@ param(
   }
 }
 
+function Run-Chocolatey {
+	Write-Debug "Running Chocolatey..."
```
Comment (ferventcoder) [inline nuget/chocolatey/tools/chocolateysetup.psm1]: Do we call write-debug elsewhere in here or does it have another name?

Comment (ferventcoder) [reply nuget/chocolatey/tools/chocolateysetup.psm1]: Found one at https://github.com/chocolatey/choco/pull/1507/files#diff-98d6abdebfea2b377e7a6d36d36c4cfaR591

File: nuget/chocolatey/tools/chocolateysetup.psm1 [hunk 2]
```diff
@@ -700,4 +701,17 @@ param(
...
 }
 
+function Run-Chocolatey {
+	Write-Debug "Running Chocolatey..."
+
+	try {
+		$chocoInstallationFolder = Get-ChocolateyInstallFolder
+		$chocoExe = Join-Path -Path $chocoInstallationFolder -ChildPath "choco.exe"
+    & $chocoExe | Out-Null
+    Write-Debug "Chocolatey execution completed successfully."
```
Comment (ferventcoder) [inline nuget/chocolatey/tools/chocolateysetup.psm1]: tabs versus spaces. Also ensure all line endings are CRLF

File: nuget/chocolatey/tools/chocolateysetup.psm1 (hunk 1 above)
Comment (gep13) [reply nuget/chocolatey/tools/chocolateysetup.psm1]: Yeah, I borrowed `Write-Debug` from elsewhere 😄

File: nuget/chocolatey/tools/chocolateysetup.psm1 [hunk 3]
```diff
@@ -700,4 +701,17 @@ param(
   }
 }
 
+function Invoke-Chocolatey {
+  Write-Debug "Running Chocolatey..."
```
Comment (ferventcoder) [inline nuget/chocolatey/tools/chocolateysetup.psm1]: `Initializing Chocolatey files, etc by running Chocolatey...`

**Pull request merged**